nncf/quantization/algorithms/weight_compression/algorithm.py

```diff
@@ -759,15 +759,13 @@ def get_statistic_points(
         model: TModel,
         graph: NNCFGraph,
         nodes_and_port_ids: Iterable[Tuple[NNCFNode, int]],
-        subset_size: Optional[int] = None,
     ) -> StatisticPointsContainer:
```

Comment:
What do you think about also removing `subset_size` parameter from `MixedPrecisionCriterion.get_statistic_points()` method for two algorithms to be more aligned? I believe `subset_size` can be provided to criterion constructor similarly as it is done for `WeightCompression` algorithm.

Reply:
Had to make it optional for data-free mixed precision, but probably it's better since the parameter is passed only once.